nnvm/src/top/tensor/reduce.cc

```diff
@@ -322,6 +322,64 @@ values over a given axis.
       topi::argmin(inputs[0], axis, param.keepdims) };
 });
 
+NNVM_REGISTER_REDUCE_OP(mean)
+  .describe(R"code(Computes the mean of array elements over given axes.
+
+Example::
+
+  data = [[[1,2],[2,3],[1,3]],
+          [[1,4],[4,3],[5,2]],
+          [[7,1],[7,2],[7,3]]]
+
+  mean(data, axis=1)
+  [3.22]
+
```

Comment:
For mean output dimension should be reduced by number of axes over which mean is calculated. Can you explain the example here?

Reply:
I'm sorry, it is a typo, fixed.